MessageContainer added

Elektri opened the pull request.

Commit:
unfinished

Commit:
Merge remote-tracking branch 'refs/remotes/origin/master' into including-material-design-icons

Commit:
Merge branch 'master' into including-material-design-icons

Commit:
unfinished

function is not working jet, ViewData can’t be used

616b2f commented:
i get this error if i try to search for a book in http://localhost:5000/Librarian/Books?q=glajlkas

![image](https://cloud.githubusercontent.com/assets/7147208/23823746/6aa7755c-0669-11e7-9663-439fd00c803b.png)

616b2f reviewed (changes requested):
- Please use 4 spaces instead of tab for intend
- I get an Error in Books/Index action (see details in conversation)

Commit:
MessageContainer added

MessageContainer added to every main View (where its needed)

set UserMessage like

this.SetMessageIntoMessageContainer("testMessage",
UserMessageType.Error);

(need to use „using Athene.Inventory.Web.Services;“)
